openstack/compute/v2/extensions/quotasets/results.go

```diff
@@ -106,3 +168,23 @@ type UpdateResult struct {
 type DeleteResult struct {
 	quotaResult
 }
+
+type quotaResultDetail struct {
+	gophercloud.Result
+}
+
+// GetResult is the response from a Get operation. Call its Extract method to interpret it
+// as a QuotaSet.
+type GetResultDetail struct {
```

Comment:
`GetDetailResult` might be better. Also the godoc comment two lines above needs updated as it still has `GetResult`.